HomeWork_03/DZ3.py

```diff
@@ -0,0 +1,79 @@
+#I love Python*42#
+
+# Zadanie 1
+x = ("I love Python")
+z = x*42
+print(z)
+
+#age_in_month#
+
+# Zadanie 2
+x = str("age_in_month")
+y = 37*12
+z = x+" "+str(y)
+print(z)
+
+#Take age in "y" Zadanie 2#
+
+# Zadanie 3
+i = str(" age_in_years")
+j = str(i)+" " + str(y//12)
+print (j)
+
+#----------------------#
+# Zadanie 4
+first_name = "Oleksandr"
+last_name = "Pitelin"
+full_name = first_name +" "+ last_name
+k = ("Му name is" +" "+ (full_name))
+l = ("I’m" + (j) + "years old")
+my_age =  k+" "+l
```

Comment:
You can avoid creation of a dozen of variables and create a string in one line, like this: `result = "My name is Oleksii, I'm " + str(age_in_years) + "Years old"